data/tools/extract-wikidata.py

```diff
@@ -0,0 +1,379 @@
+"""
+Extracts lexicographic data from Wikidata for Unicode Inflection usage
+
+First you need to download the lexicographic data wholesale from Wikidata.
+You can do this by downloading the data from the Web, with either of the
+following two commands.
+
+The first one downloads a specifc version of the data. The following
+command downloads the first version of 2025:
+
+wget -O lexemes.json.bz2 https://dumps.wikimedia.org/wikidatawiki/entities/20250101/wikidata-20250101-lexemes.json.bz2
+
+A list of all available versions can be found on the Web here:
+https://dumps.wikimedia.org/wikidatawiki/entities/
+
+Or you can download the most recent version using the following command:
```

Comment:
I would put this as a top option with a comment on how to download specific one (by date) - so essentially flip the order you have now.